scout/server/blueprints/variants/controllers.py

```diff
@@ -1007,19 +1007,19 @@ def check_form_gene_symbols(
 
     errors = {
         "non_clinical_symbols": {
-            "alert": "Gene not included in case clinical list",
+            "info": "Genes not included in case gene panels",
             "gene_list": non_clinical_symbols,
         },
         "not_found_symbols": {
-            "alert": "HGNC symbol not present in genes collection",
+            "alert": "HGNC symbols not present in database genes collection",
             "gene_list": not_found_symbols,
         },
         "not_found_ids": {
-            "alert": "HGNC id not present in genes collection",
+            "alert": "HGNC ids not present in database genes collection",
             "gene_list": not_found_ids,
         },
         "outdated_symbols": {
-            "alert": "Clinical list contains a panel with an outdated symbol for genes",
+            "alert": "Case gene panels contain an outdated symbol for genes",
```

Comment:
Maybe this should also stay on the info stage? It could really be from a panel that the user does not maintain, and it can be a bit slow finding the panel in question.